app/js/FullPage.js

```diff
@@ -41,26 +41,26 @@ export class FullPage {
     let that = this;
```

Comment:
If you just change your anonymous functions to arrowFunctions this "let that = this;" will not be necessary anymore.